.github/workflows/github.yml

```diff
@@ -130,7 +130,7 @@ jobs:
           ${{github.workspace}}/**/${{ env.VCMI_PACKAGE_FILE_NAME }}.${{ matrix.extension }}
 
     - name: Upload build
-      if: ${{ matrix.pack == 1 && github.ref == 'refs/heads/develop' && matrix.platform != 'msvc' }}
+      if: ${{ matrix.pack == 1 && (github.ref == 'refs/heads/develop' || startsWith(github.ref, 'refs/heads/features/')) && matrix.platform != 'msvc' }}
```

Comment:
@nullkiller , if we go with same approach (feature branch), we can keep it. Otherwise will be reverted

Reply:
A new branch was created and the PR was retargeted